docs/conf.py

```diff
@@ -0,0 +1,159 @@
+
+project = 'PowSyBl Metrix'
+github_repository = "https://github.com/powsybl/powsybl-metrix/"
+copyright_year = f'2018-{ datetime.datetime.now().year }'
+
+# Find the version and release information.
+# We have a single source of truth for our version number: pip's __init__.py file.
+# This next bit of code reads from it.
+file_with_version = os.path.join(source_path, "pom.xml")
+with open(file_with_version) as f:
+    next_line_contains_version = False
+    for line in f:
+        if next_line_contains_version == False:
+            m = re.match(r'^ {4}\<artifactId\>powsybl\-metrix\<\/artifactId\>', line)
+            if m:
+                next_line_contains_version = True
+        else:
+            m = re.match(r'^ {4}\<version\>(.*)\<\/version\>', line)
+            if m:
+                __version__ = m.group(1)
+                # The short X.Y version.
+                version = ".".join(__version__.split(".")[:2])
+                # The full version, including alpha/beta/rc tags.
+                release = __version__
+                break
+    else:  # AKA no-break
+        version = release = "dev"
+
+# Short project name
+short_project = project.split(" ", 1)[1] if project.split(" ", 1)[0] == "PowSyBl" else project
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc',
+              'sphinx.ext.autosummary',
+              'sphinx.ext.viewcode',
+              'sphinx.ext.doctest',
+              'sphinx.ext.napoleon',
+              'sphinx.ext.todo',
+              'sphinx.ext.intersphinx',
+              'sphinx_tabs.tabs',
+              'myst_parser',
+              # Extension used to add a "copy" button on code blocks
+              'sphinx_copybutton']
+myst_enable_extensions = [
+    "amsmath",
+    "colon_fence",
+    "dollarmath",
+    "attrs_inline"
+]
+myst_heading_anchors = 6
+
+# Add any paths that contain templates here, relative to this directory.
+templates_path = ['_templates']
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
+
+
+# -- Options for HTML output -------------------------------------------------
+
+# The theme to use for HTML and HTML Help pages.  See the documentation for
+# a list of builtin themes.
+#
+html_theme = "furo"
+
+html_title = f"{project} v{release}"
+html_short_title = f'{short_project} v{release}'
+
+html_logo = '_static/logos/logo_lfe_powsybl.svg'
+html_favicon = "_static/favicon.ico"
+
+html_context = {
+    "copyright_year": copyright_year,
+    "github_repository": github_repository,
+    "sidebar_logo_href": "http://powsybl-metrix.readthedocs.io/"
```

Comment:
I think that the link should be: https://powsybl.readthedocs.org so that when you click on the logo on the top-left corner, you are redirected to the powsybl "main" readthedocs.

Reply:
I'm not sure about this, because it would mean the user has no way to directly get back to the index of the current readthedocs, right?
Would it not be better to add a link to powsybl-core somewhere?